tests/omise/TestConfig.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-define('OMISE_PUBLIC_KEY', 'pkey');
-define('OMISE_SECRET_KEY', 'skey');
+define('OMISE_PUBLIC_KEY', 'pkey_test_4zfuolwdegumf4k2qba');
+define('OMISE_SECRET_KEY', 'skey_test_4zfuom5ugbndd4z8nlf');
```

Comment:
Ok, I tried to remove this file from test case file and run test again, It's fine.
I think it not necessary for now (but not anymore, I kept it for test with server sometimes).

I'll clean my pkey, skey from this file to `pkey`, `skey` (default) but I think it should keep this file in tests folder not remove it.